test/01_contraction/contraction_test.cpp

```diff
@@ -656,13 +659,36 @@ namespace hiptensor
             auto reference = resource->allocDevice(sizeD);
             resource->copyData(reference, resource->hostD(), sizeD);
 
+            // Compute tolerance based on compute type
+            auto dimension = a_ms_ks.mLengths.size() / 2;
+            auto nelems_k = std::accumulate(a_ms_ks.mLengths.begin() + dimension,
+                                            a_ms_ks.mLengths.end(),
+                                            size_t{1},
+                                            std::multiplies<size_t>());
+
+            auto eps = getEpsilon(computeType);
```

Comment:
I noticed that CK only use epsilon of f32 or f64 to generate the tolerance. 
https://github.com/ROCm/composable_kernel/blob/b9ba5b26769292827cd65d3be221b4b0bdb5d4b8/profiler/include/profiler/profile_contraction_impl.hpp#L142-L143

It makes sense to me since epsilon of bf16 (0.0078) is too large for this formular. 

double tolerance = 2 * nelems_k * eps  

For example, when nelems_k  == 1000, tolerance is 7.8

We can follow the same rule as CK :)